controllers/util/util.go

```diff
@@ -11,8 +11,8 @@ import (
 )
 
 var (
-	SuccessDel = flag.Duration("job-cleanup-on-success-delay", 0, "Seconds to delay job deletion after successful runs. 0 means no delay")
-	ErrDel     = flag.Duration("job-cleanup-on-error-delay", time.Hour*24, "Seconds to delay job deletion after errored runs. 0 means no delay")
+	SuccessDel = flag.Duration("job-cleanup-on-success-delay", 0, "Duration to delay job deletion after successful runs. 0 means no delay")
```

Comment:
What is the default unit?

Reply:
By default time.Duration is nanoseconds if the user doesn't provide a unit with their input. I can add that to the description as well.

Reply:
this can be a follow up if you want

Reply:
we changed it from seconds to duration in #308 for consistency. Should we revert it back?